_posts/2016-06-20-user-testing.md

```diff
@@ -0,0 +1,149 @@
+- How many users access the site on desktop, tablet, and mobile?
+  - Audience <i class="fa fa-long-arrow-right"></i> Mobile <i class="fa fa-long-arrow-right"></i> Overview
+- How do users get to the site?
+  - Acquisition <i class="fa fa-long-arrow-right"></i> All Traffic <i class="fa fa-long-arrow-right"></i> Channels
+- Which pages are most viewed?
+  - Behavior <i class="fa fa-long-arrow-right"></i> Site Content <i class="fa fa-long-arrow-right"></i> All Pages
+- What are the top exit pages, and which of them are highly viewed pages?
+  - Behavior <i class="fa fa-long-arrow-right"></i> Site Content <i class="fa fa-long-arrow-right"></i> Exit Pages
+
+Reviewing these item with our goals and users in mind informs the design of the user testing to follow. We can focus on the parts of the site causing the most users to leave and the behaviors we want to change, and we can test on the devices important to the users.
+
+### Competitive analysis
+
+In tandem with reviewing analytics, we examine our client's competitors' websites (typically with a list of competitors/peers provided by the client). By reviewing the navigation, layout, functionality, and user experience of websites with goals similar to our client's, we can ascertain critical feedback:
+
+- What are other sites doing well? What improvements could be applied to our client's site?
+- Where are other sites falling short? How can we ensure these bases are covered on our client's site?
+- What user experience problems exist on other sites? Can we test for these same issues on our client's site during the user testing phase?
+- How does our client's site compare overall in terms of quality? How can we elevate our client's site among its peers?
+
+### User testing
+
+After gathering information from analytics and competitive analysis, we design our user tests. These are in-person meetings with a single participant at a time who is asked to complete tasks on the website, acting as one of the user personas. After reading the persona description and the task to the user, we observe their experience and record how they interact with the site.
+
+##### Designing the tests
+
+Designing the user tests is a multi-step process:
+
+1. Decide which personas will be tested and how many participants are needed. User personas to be tested are chosen based on the client's priority and potential/known issues found in the review of analytics and competitors. Number of participants can be surprisingly small and still quite effective, as per the Nielson Group's [detailed analysis](https://www.nngroup.com/articles/why-you-only-need-to-test-with-5-users/). Ideally user testing would be iterative -- a few users would be tested, followed by improvements to the site targeting the results of those tests, followed by more testing, etc. If this isn't possible, five participants with one round of testing is typically effective.
```

Comment:
An alternative to testing after the "improvement" has been made could be to simultaneously A/B test different feature/layout/call-to-actions scenarios with different users during the same testing session.